src/components/EndorsementChain/EndorsementChainLayout/EndorsementChainLayout.test.tsx

```diff
@@ -1,9 +1,14 @@
+import { useIdentifierResolver } from "@govtechsg/address-identity-resolver";
 import { act, fireEvent, render, screen } from "@testing-library/react";
 import React from "react";
 import { TitleEscrowEvent, TradeTrustErc721Event } from "../../../types";
 import { EndorsementChainLayout } from "./EndorsementChainLayout";
 
-jest.mock("../../../common/hooks/useIdentifierResolver");
+jest.mock("@govtechsg/address-identity-resolver", () => ({
```

Comment:
Same as Entrypoint Test file

Reply:
Actually what's the purpose of mocking here, hook is not used in the component?

Reply:
its used in another component that is used in this component, thus we need to mock it.